### PR TITLE
Cleanup graphics to support higher scaled images

### PR DESCRIPTION
**Pull request by JamesVanBoxtel**
Fixed some spots in graphics that had non pixel aligned positions
Fixed a bug with telegraph position for high resolution images
Fixed frame, wall, match type, and pause menu rendering for higher resolutions
Removed csprnttests as it was periodically failing and not a real practical test
upgraded cursor name to make it less confusing, still falls back to the old one

### CODE REVIEW

**Comment by Shosoul** [comment]
Idea: assuming the cursor sizes remain the same, just have the game rename the cursor on its own

**Comment by JamesVanBoxtel** [comment]
> Idea: assuming the cursor sizes remain the same, just have the game rename the cursor on its own

This is really more to help the modding community in the future not get confused. When I first made a cursor mod I thought I was making separate ones for player 1 and 2

**Comment by Shosoul** [comment]
> > Idea: assuming the cursor sizes remain the same, just have the game rename the cursor on its own
> 
> This is really more to help the modding community in the future not get confused. When I first made a cursor mod I thought I was making separate ones for player 1 and 2

True, though it will also help them if legacy mods are also correct as well if they try to use a legacy mod for reference

**Comment by JamesVanBoxtel** [comment]
Oh you are saying literally have the game modify the file names? IDK I think its better just to update the base mods and documentation.

File: graphics_util.lua [hunk 1]
```diff
@@ -36,6 +46,31 @@ function load_img_from_supported_extensions(path_and_name)
...
+  end
+end
+
+-- Draws a image at the given screen spot with the given width. Scaling to keep the ratio.
+function GraphicsUtil.drawScaledWidthImage(image, x, y, width)
+  if image ~= nil and x ~= nil and y ~= nil and width ~= nil and height ~= nil then
+    local scaleX = width / image:getWidth()
+    GraphicsUtil.drawImage(image, x, y, scaleX, scaleX)
+  end
+end
```
Comment (Endaris) [inline graphics_util.lua:72]: Don't need height if it's not a param.

**Review by Endaris — COMMENTED** [review]
Only code review, didn't really test so far.

**Comment by JamesVanBoxtel** [comment]
Okay @Shosoul this is all merged up now and ready for review again.

**Review by Shosoul — APPROVED** [review]
This is fine, but I might write a follow-up pr that will rename the images for the player

**Pull request merged**